src/mmds/src/data_store.rs

```diff
@@ -34,11 +40,21 @@ impl Default for Mmds {
         Mmds {
             data_store: Value::default(),
             is_initialized: false,
+            response_format: ResponseFormat::Imds,
         }
     }
 }
 
 impl Mmds {
+    /// Response format setter.
+    pub fn set_json_format(&mut self, is_json: bool) {
```

Comment:
To be consistent with the naming, we should either name this `set_response_format` and send the `ResponseFormat` as a parameter, or name it `set_json_response_format` in which case we don't need to send the bool as a parameter.